attendance/API_HR/c-hr.php

```diff
@@ -5538,6 +5538,51 @@ public static function getInventoryAuditStatusforYearMonth( $inventory_id, $year
         return $return;
     }
 
+    public static function getInventoriesAuditStatusForYearMonth( $month, $year ){
+
+        $return = array();
+        $error = 0;
+        $message = "";
+
+        $q = "SELECT 
+        machines_list.id, 
+        machines_list.machine_type, 
+        machines_list.machine_name, 
+        files.file_name, 
+        inventory_audit_month_wise.id as audit_id, 
+        inventory_audit_month_wise.inventory_id, 
+        inventory_audit_month_wise.month, 
+        inventory_audit_month_wise.year, 
+        inventory_comments.comment 
+        FROM 
+        machines_list 
+        left join files on machines_list.file_inventory_photo = files.id 
+        left join inventory_audit_month_wise on machines_list.id = inventory_audit_month_wise.inventory_id 
+        AND inventory_audit_month_wise.month = $month 
+        AND inventory_audit_month_wise.year = $year 
+        left join inventory_comments on inventory_audit_month_wise.inventory_comment_id = inventory_comments.id 
+        ORDER BY id DESC";
+
+        $runQuery = self::DBrunQuery($q);
+        $rows = self::DBfetchRows($runQuery);
+
+        if (sizeof($rows) == 0) {
+            $error = 1;
```

Comment:
error  should be 0

error will be 1 in case if we have to restrict or on error. ( see other api's result you will get to know )